src/pip/_internal/locations.py

```diff
@@ -75,16 +75,16 @@ def get_src_prefix():
 except AttributeError:
     user_site = site.USER_SITE
 
+bin_user = sysconfig.get_path('scripts')
+
 if WINDOWS:
     bin_py = os.path.join(sys.prefix, 'Scripts')
-    bin_user = os.path.join(user_site, 'Scripts')
```

Comment:
This variable is needed elsewhere in the tree (`src/pip/_internal/req/req_uninstall.py`)

Reply:
I didn't remove it, I just moved it up.